client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -242,13 +252,17 @@ const Checkout = React.createClass( {
 			} );
 		}
 
+		if ( ! selectedSiteSlug ) {
```

Comment:
Again, I think kicking back to `/checkout/` here is less than ideal, could be covered in the issue I mentioned at line 243